### DISCUSSION TITLE
善用Windows任务计划程序，实现maa定时自启动

### DISCUSSION THREAD

**Discussion by sunflower126** [MaaAssistantArknights/MaaAssistantArknights#9421] [General]
发现可以使用Windows任务计划程序定时启动maa和计算机，然后再通过maa启动模拟器和明日方舟，任务计划触发器可以设置多个时间，建议开发者在maa帮助文档里加上一点这方面的内容介绍
![maa触发器](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/146410014/011a109b-80b5-4711-9275-6a710d256a13)
![自启动操作](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/146410014/990bd2e0-314a-4fb1-bc3f-87ca9e4c828e)
![参考条件](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/146410014/30a1ba3a-e472-4782-98ed-22247a26f8af)

**Comment by Constrat** [comment]
If a user needs more complicated ways to schedule MAA, it will already know of the existence of the Task Scheduler.
![image](https://github.com/user-attachments/assets/282349a1-6f6d-47c2-8f92-7e73c3328578)
As for the generic user these are already enough.
In case it really needs more complicated schedules without using Task Scheduler, you can always use two or even three MAAs at a time.
